internal/metrics/metricsManager_test.go

```diff
@@ -0,0 +1,91 @@
+package metrics
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/google/uuid"
+	"github.com/openshift/assisted-service/models"
+	"github.com/prometheus/client_golang/prometheus"
+
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Metrcis Manager", func() {
+
+	const (
+		clusterVersion = "4.6"
+		emailDomain    = "redhat.com"
+	)
+
+	var (
+		mm        *MetricsManager
+		clusterID strfmt.UUID
+	)
+
+	BeforeEach(func() {
+		clusterID = strfmt.UUID(uuid.New().String())
+		prometheusRegistry := prometheus.DefaultRegisterer
+		mm = NewMetricsManager(prometheusRegistry)
+	})
+
+	AfterEach(func() {
+	})
+
+	Context("Validation metrics", func() {
+
+		It("Host validation metrics", func() {
```

Comment:
Maybe table driven will be clearer?

Reply:
Isn't a table driven will make all tests run in the same `It` ?
I don't want that because I want a new cluster for each test - otherwise it will be a lot harder to filter only the metrics generated by the specific test (by filtering clusterID).

Reply:
Table driven should run in a different `It` for example https://github.com/openshift/assisted-service/blob/master/internal/oc/release_test.go#L156-L159

Reply:
Done.